app/views/ingredients/index.html.erb

```diff
@@ -0,0 +1,4 @@
+<% @ingredients.each do |ingredient| %>
+<p><%= ingredient.name %>: <%= ingredient.cost %></p>
+<% end %>
+<%= link_to 'Sort Ingredients Alphabetically', "/ingredients?sort=yeahbaby" %>
```

Comment:
This does accomplish what the _spirit_ of the user story asks for, but it does just ask for the list to be ordered alphabetically, so we don't need a link that does it. You can alter the `@ingredients` on line 1 to always be ordered after implementing this user story by calling it from the controller, which wouldn't need this query param.